packages/synapse-react-client/src/components/DownloadCart/DownloadDetails.tsx

```diff
@@ -17,13 +18,22 @@ export default function DownloadDetails(props: DownloadDetailsProps) {
   const iconClassName = isInactive ? 'SRC-inactive' : 'SRC-primary-text-color'
   return (
     <span className="DownloadDetailsV2">
-      <span className="item">{!isInactive && <> {numFiles} Files </>}</span>
+      <span className="item">
+        {!isInactive && (
+          <>
+            {numFiles.toLocaleString()} {pluralize('File', numFiles)}
+          </>
+        )}
+      </span>
       <span className="item">
         <span className={iconClassName}>
-          <IconSvg icon="packagableFile" />
+          <IconSvg wrap={false} icon="packagableFile" />
```

Comment:
Use `wrap={false}` to remove a `span` wrapper that our IconSvg class automatically adds to fix vertical alignment. This is some legacy debt; I want the wrapper to not be included by default, but that would require many changes that haven't happened yet. I would not have expected you to know this!